q2mm/compare.py

```diff
@@ -80,13 +85,17 @@ def pretty_data_comp(r_data, c_data, output=None, doprint=False):
             else:
                 score_typ[c.typ + '-o'] += score
         strings.append('  {:<30}  {:>7.2f}  {:>11.4f}  {:>11.4f}  {:>11.4f}  '\
-                       '{:>5}  '.format(
+                       '{!s:>5}  '.format(
```

Comment:
I didn't know about explicit conversion flags until now.

Reply:
Yeah haha, I had to learn that one because the string.format() method doesn't like NoneTypes now so it had to explicitly convert to string there.